src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    const { title, description } = movie;
+    const stringToSearch = `${title} ${description}`;
+
+    return stringToSearch.toLowerCase()
+      .trim()
+      .includes(query.trim().toLowerCase());
+  });
```

Comment:
It's probably just me but it looks complicated to use variables in the function expression which is already corresponding to another variable. 
The code below looks easier to read in my opinion.

```
  const visibleMovies = moviesFromServer.filter(movie => (
    ifIncludes(movie.title) || ifIncludes(movie.description)));
```

Reply:
it could be either way - both are fine